web/server/README.md

```diff
@@ -101,6 +101,24 @@ openssl req -newkey rsa:2048 -nodes -sha512 -x509 -days 365 -keyout key.pem -out
 > openssl speed rsa2048 rsa4096
 > ```
 
+### Select TLS library
+
+Netdata can be compiled against either `OpenSSL` or `WolfSSL` libraries, but only `OpenSSL` gives support for all
+Netdata features. For more details, take a look on the table:
+
+| Netdata feature | OpenSSL |WolfSSL|
+|:---------:|:-----------:|:------------:|
+| HTTPS protocol | Available|Available|
+| Encrypted stream| Avaiable|Available|
+| memory mode = `dbengine`| Avaiable|Available|
+| HTTPS on exporter| When destination accepts|When destination accepts, but with a worse performance when compared with `OpenSSL`. <sup id="a1">[1](#f1)</sup>|
+| Netdata cloud| Available| `not Available` due problems with `libwebsockets`. <sup id="a2">[2](#f2)</sup> |
+<b id="f1">1</b> When testing with InnoDB, we observed error mesages like `"tls: oversized record received with length
+21536"`. [↩](#a1)
+<b id="f2">2</b> When `libwebsockets` is compiled against `WolfSSL` it says that the function
+`X509_VERIFY_PARAM_set1_host` is not present, but `WolfSSL` has the function `wolfSSL_X509_VERIFY_PARAM_set1_host` that
+is not used. [↩](#a2)
```

Comment:
This shouldn't be inside the table.

Reply:
Good catch. @thiagoftsm, I think you can just add an empty line between the table and the start of the footnotes to break them out of the table.

Reply:
Thank you guys, I fixed the problem.
@vlvkobal I also fixed the compilation problem with distributions using old SSL versions.